DQM/TrackingMonitor/src/TrackAnalyzer.cc

```diff
@@ -850,6 +853,21 @@ void TrackAnalyzer::bookHistosForBeamSpot(DQMStore::IBooker& ibooker) {
     DistanceOfClosestApproachToBS->setAxisTitle("Track d_{xy} wrt beam spot (cm)", 1);
     DistanceOfClosestApproachToBS->setAxisTitle("Number of Tracks", 2);
 
+    if (Folder == "Tr") {
```

Comment:
@hbecerri may I ask you to remind me what you mean by this check ?

Reply:
Hello @mtosi, as is mentioned before in this PR, the modules I modified are being used by other subsystems (MUO, HLT/EGM and HLT/EXO) so the new plots were also in different workspaces other than Tracking. So the check is to create the new plots only for Tracking.